startup.s

```diff
@@ -0,0 +1,43 @@
+	.section .exception
+	.globl _start
+	.type _start, function
+
+/* Boot code at 0xbfc00000 */
+_start:	la	$sp, _estack	# Set stack pointer.
+	la	$gp, _gp	# Prepare global pointer.
+
+	/* Copy .data from ROM to RAM.
+	 * .data is located in ROM just after .text, so it starts at _etext.
+	 * It should land at the beggining of RAM, which is pointed at by _data.
+	 * Copy words one by one, until _edata is reached. */
+	la	$t1, _data	# dest = _data
+	la	$t2, _edata	# dest_end = _edata
+	la	$t3, _etext	# src = _etext
+copy_next_word:
```

Comment:
I would organize this code in a different way w.r.t. symbol names and their use.

https://sourceware.org/binutils/docs/as/Symbol-Names.html

I propose to give a symbol `cope_rom_to_ram` to the beginning of the loop  and then use local label `.Lnext_word` or similar. Local labels are very handy tool in assembly.